wazo_confd/plugins/meeting/resource.py

```diff
@@ -190,20 +184,6 @@ def delete(self, uuid):
         self.service.delete(model)
         return '', 204
 
-    def _find_user_uuid(self):
-        token = request.headers.get('X-Auth-Token') or request.args.get('token')
```

Comment:
This is not the default behavior of `AuthVerifier`: it will only look at the header, not the query string.
`AuthVerifier` needs to be instanciated with `extract_token_id = extract_token_id_from_query_or_header`. This will introduce a slight change in behavior: the token from query string will take precedence over the header.

Reply:
Actually we don't need the `token=` query string parameter so it's OK to remove it.